messages/bitbox02_system.proto

```diff
@@ -26,11 +26,13 @@ message DeviceInfoRequest {
 }
 
 message DeviceInfoResponse {
-    string name = 1;
-    bool initialized = 2;
-    string version = 3;
-    bool mnemonic_passphrase_enabled = 4;
-    uint32 monotonic_increments_remaining = 5;
+  string name = 1;
+  bool initialized = 2;
+  string version = 3;
+  bool mnemonic_passphrase_enabled = 4;
+  uint32 monotonic_increments_remaining = 5;
+  // From v9.6.0: "ATECC608A" or "ATECC608B".
+  string securechip_model = 6;
```

Comment:
I wonder if an enum would've worked better here?

Reply:
I wonder the same. If it is an enum, extending it means all client libraries have to be updated with human readable strings matching the enum types.

Like this, the string can be shown as it, and no further protobuf message and client library changes are needed should the other models be added.

Reply:
Yeah, I guess if the main usecase is to show it in a UI, which it is, string is probably better :+1: 
I think the field comment with exact strings is enough if anyone will want to base logic on a specific model.